amoro-common/src/main/java/org/apache/amoro/process/TableProcessState.java

```diff
@@ -53,7 +53,7 @@ public long getId() {
   }
 
   public String getName() {
-    return action.getDescription();
```

Comment:
Action must be annotated with @StateField, otherwise Process recovery cannot be performed

Reply:
I don't think @StateField is what you meant.
Please check it